pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+
+class TestBodies():
+
+    def test_body_init(self):
+        body = Body('body')
+
+        assert body.name is None
+        assert body.mass == Symbol('mass')
+
+        sys = System(method='joints')
+        sys.add_body(body)
+
+        assert body.name == 'body_1'
```

Comment:
The idea is that if user does not passes a name, it automatically increments and creates a unique body name which is required to define other attributes of the body.